app/Console/Command/ChangePlaceTypeOnEvents.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Console\Command;
+
+use Broadway\CommandHandling\CommandBus;
+use CultuurNet\UDB3\Event\Events\AvailableFromUpdated;
+use CultuurNet\UDB3\Offer\Commands\UpdateType;
+use CultuurNet\UDB3\Search\ResultsGenerator;
+use CultuurNet\UDB3\Search\SearchServiceInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Logger\ConsoleLogger;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+final class ChangePlaceTypeOnEvents extends AbstractCommand
+{
+    private SearchServiceInterface $searchService;
+
+    private int $errors;
+
+    private int $success;
+
+    public function __construct(
+        CommandBus $commandBus,
+        SearchServiceInterface $searchService
+    ) {
+        $this->searchService = $searchService;
+        $this->errors = 0;
+        $this->success = 0;
+        parent::__construct($commandBus);
```

Comment:
```suggestion        
        parent::__construct($commandBus);
        $this->searchService = $searchService;
        $this->errors = 0;
        $this->success = 0;
```

It seems more logical to first call the parent constructor